src/ImageSharp/Common/ByteOrder.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Six Labors.
+// Licensed under the Apache License, Version 2.0.
+
+namespace SixLabors.ImageSharp
+{
+    /// <summary>
+    /// The tiff data stream byte order enum.
```

Comment:
The description should not be specific to Tiff.

Reply:
changed with b0e965fd